cristin-commons/src/main/java/no/unit/nva/cristin/model/SearchResponse.java

```diff
@@ -28,7 +30,7 @@
     JsonPropertyNames.CONTEXT, JsonPropertyNames.ID, JsonPropertyNames.SIZE, JsonPropertyNames.SEARCH_STRING,
     JsonPropertyNames.PROCESSING_TIME, JsonPropertyNames.FIRST_RECORD, JsonPropertyNames.NEXT_RESULTS,
     JsonPropertyNames.PREVIOUS_RESULTS, JsonPropertyNames.HITS})
-public class SearchResponse {
+public class SearchResponse<E> {
```

Comment:
We can have unordered but then we have to spend some time creating equals and hashcode and then update unit tests to compare on object level instead of json level. I also ran into problems comparing generic types like List<?> because its somehow converted to another type (LinkedHashMap) so I had to use Object Mapper to convert it into a specific type.

`List<Person> actualPersons = OBJECT_MAPPER.convertValue(actual.getHits(), new TypeReference<>() {});`